compiler/qsc_passes/src/lib.rs

```diff
@@ -36,6 +36,9 @@ use qsc_rca::{PackageComputeProperties, PackageStoreComputeProperties};
 use replace_qubit_allocation::ReplaceQubitAllocation;
 use thiserror::Error;
 
+pub(crate) static CORE_NAMESPACE: &[&str] = &["Microsoft", "Quantum", "Core"];
+pub(crate) static QIR_RUNTIME_NAMESPACE: &[&str] = &["QIR", "Runtime"];
```

Comment:
I see there are a few spots where we do `.find_namespace(WELL_KNOWN_NAMESPACE.iter().copied())` . Feels like a lot of work when the namespace ID is constant each time. Can we maybe cache the found namespace ID with these statics?

Reply:
Alternatively, these namespaces could be extra special, and their IDs can be exposed directly from `Table` ?

Reply:
Yeah... we're getting into diminishing returns. 

If we're doing _anything_ about it, I'd literally hardcode namespace IDs `1` and `2` for these well known namespaces, and just insert them into the `Table` at creation time. Anything more complex than that would be silly.

But even that.... probably not worth the trouble at the moment.